README.md

```diff
@@ -571,6 +572,22 @@ type Result = RequiredKeys<T>;
 // "a" | "c"
 ```
 
+### PickKeys
+
+Gets keys of properties of given type in object type.
+
+```typescript
+type T = {
+  a: number;
+  b?: string;
+  c: string | undefined;
+  d: string;
+};
+type Result = PickKeys<T, string>;
+// Result:
+// "d"
```

Comment:
let's add another example with a complex type to better illustrate the possibilities
```suggestion
type Result = PickKeys<T, string>;
// Result: "d"
type Result2 = PickKeys<T, string | undefined>;
// Result2: "b" | "c" | "d"
```

Reply:
done